pkg/proxyserver/controller/controller.go

```diff
@@ -179,7 +179,7 @@ func (c *ProxyServiceInfoController) syncHandler(key string) error {
 	return nil
 }
 
-var optionsKey = []string{"service", "port", "path", "sub-resource", "secret"}
+var optionsKey = []string{"service", "port", "path", "sub-resource", "secret", "caConfigMap"}
```

Comment:
Also update the documentation here: https://github.com/open-cluster-management/multicloud-operators-foundation/blob/main/docs/proxyserver/proxyserver.md#configuration-example